common/src/main/java/org/apache/celeborn/common/protocol/RpcNameConstants.java

```diff
@@ -31,6 +31,6 @@ public class RpcNameConstants {
   public static String WORKER_EP = "WorkerEndpoint";
 
   // For Driver(RssShuffleManager)
-  public static String RSS_METASERVICE_EP = "RssMetaServiceEndpoint";
-  public static String RSS_METASERVICE_SYS = "RssMetaSys";
+  public static String METASERVICE_EP = "MetaServiceEndpoint";
```

Comment:
It's clear that our meta is LifecycleManager and it's engine independent. Why not change it to LIFECYCLEMANAGER_EP? There is no need to add a new concept here.

Reply:
DONE